teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -442,6 +442,7 @@ interface Build {
     val buildProblems: Sequence<BuildProblemOccurrence>
 
     fun addTag(tag: String)
+    fun addComment(comment: String)
```

Comment:
Am I right that we cannot have several comments for a build, so adding a new comment will overwrite the previous one? If so it would be better to name it `setComment` instead.

Reply:
done